internal/controller/validationresult_controller.go

```diff
@@ -66,16 +97,66 @@ func (r *ValidationResultReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			)
 		}
 
-		// TODO: send result to a sink
+		// emit ValidationResult to a sink - either on the 1st reconciliation, or if its hash changes
+
+		if vc.Spec.Sink != nil && len(vr.Status.Conditions) > 0 && (!ok || prevHash != currHash) {
+			sinkState = v1alpha1.SinkEmitFailed
+
+			sink := sinks.NewSink(vc.Spec.Sink.Type, r.Log)
```

Comment:
Not something for now, but it could be useful to set this up so that we can configure multiple sinks for a single validator. that way we can get notified of errors on slack, palette, or any other sinks (pagerduty, datadog, etc.)